src/lib/util/Config.ts

```diff
@@ -73,7 +75,7 @@ class TypeChecker {
 
 	/* eslint-disable @typescript-eslint/no-explicit-any */
 	booleanCheck = (value: any, propertyName: string, engine: ENGINE): boolean => {
-		if (value instanceof boolean) {
+		if (typeof value === 'boolean') {
```

Comment:
This was comparing to an oclif type previously, the booleanCheck wasn't used by any other code.